dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/message/LengthFieldStreamingDecoder.java

```diff
@@ -46,8 +45,6 @@
 
     private int requiredLength;
 
-    private InputStream dataHeader = new ByteArrayInputStream(new byte[0]);
-
     public LengthFieldStreamingDecoder() {
         this(4);
     }
```

Comment:
**Performance Issue**: The 'dataHeader' InputStream field is declared but never used in the 'LengthFieldStreamingDecoder' class. Unused fields can lead to confusion and should be removed if they are not planned to be used. <br> **Fix**: Remove the 'dataHeader' field as it is not used anywhere in the class. <br> **Code Suggestion**: 
 ```
 -    private InputStream dataHeader = new ByteArrayInputStream(new byte[0]);
 ```